doonut-core/src/main/java/com/doonutmate/doonut/member/entity/MemberEntity.java

```diff
@@ -58,4 +58,9 @@ public class MemberEntity extends BaseTimeEntity {
     public void delete() {
         deleted = true;
     }
+
+    public void updateNameOrProfileImage(String name, List<ProfileImageEntity> newProfileImages) {
```

Comment:
`updateNameAndProfileImage`가 좀 더 적절할 것 같네요!

```suggestion
    public void updateNameAndProfileImage(String name, List<ProfileImageEntity> newProfileImages) {
```